Example/Example Swift/ComponentAPICoordinator.swift

```diff
@@ -409,15 +409,22 @@ extension ComponentAPICoordinator: CameraViewControllerDelegate {
         validate([document]) { result in
             switch result {
             case .success(let documentRequests):
-                self.documentRequests.append(contentsOf: documentRequests)
-                self.upload(documentRequests: documentRequests)
-                
-                if self.giniConfiguration.multipageEnabled, self.documentRequests.count > 1 {
-                    if let imageDocument = document as? GiniImageDocument {
+                if let qrCodeDocument = document as? GiniQRCodeDocument {
+                    viewController.showPopup(forQRDetected: qrCodeDocument) {
```

Comment:
Did you consider showing the popup before invoking the `didCapture` method.
I really think it should be part of the camera view controller's responsibilities to return a QR code only after it has been confirmed. It would also make the flow of the code clearer.

Reply:
I decided to use this approach because the validation responsibility should be unified outside of the camera screen. 
What I may consider is to validate the QR codes before delivering them to the camera, seeing if it is worth in terms of performance.
Do you what me to tackle this in a different PR?

Reply:
#242